src/components/ConfirmModal.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+    TouchableOpacity, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Header from './Header';
+import Modal from './Modal';
+import styles from '../styles/styles';
+import CONST from '../CONST';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+
+const propTypes = {
+    /** Title of the modal */
+    title: PropTypes.string.isRequired,
+
+    /** A callback to call when the form has been submitted */
+    onConfirm: PropTypes.func.isRequired,
+
+    /** A callback to call when the form has been closed */
+    onCancel: PropTypes.func.isRequired,
+
+    /** Modal visibility */
+    isVisible: PropTypes.bool.isRequired,
+
+    /** Confirm button text */
+    confirmText: PropTypes.string,
+
+    /** Cancel button text */
+    cancelText: PropTypes.string,
+
+    /** Modal content text */
+    prompt: PropTypes.string,
+
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    confirmText: 'Yes',
+    cancelText: 'No',
+    prompt: '',
+};
+
+const ConfirmModal = props => (
+    <Modal
+        onSubmit={props.onConfirm}
+        onClose={props.onCancel}
+        isVisible={props.isVisible}
+        type={props.isSmallScreenWidth
+            ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED
+            : CONST.MODAL.MODAL_TYPE.CONFIRM}
+    >
+        <View style={styles.m5}>
+            <View style={styles.flexRow}>
+                <Header title={props.title} />
+            </View>
+
+            <Text style={[styles.textLabel, styles.mt4]}>
```

Comment:
Can we use `styles.textP` here please?

Reply:
Also, this is not a blocker but I think I prefer adding margin to the bottom of the modal header as opposed to the top of this text block.